visualization/src/interactor_style.cpp

```diff
@@ -38,6 +38,7 @@
 
 #include <list>
 #include <pcl/common/angles.h>
+#include <pcl/visualization/boost.h>
```

Comment:
Missing `// for` here

Reply:
Split into more specific includes and added explanation